### ISSUE TITLE
 BUILD FAILURE

### ISSUE THREAD

**Issue by bingfenglai** [apache/doris#12212]
Plugin net.sourceforge.czt.dev:cup-maven-plugin:1.6-cdh or one of its dependencies could not be resolved: Failed to read artifact descriptor for net.sourceforge.czt.dev:cup-maven-plugin:jar:1.6-cdh: Could not transfer artifact net.sourceforge.czt.dev:cup-maven-plugin:pom:1.6-cdh from/to cloudera-public (https://repository.cloudera.com/artifactory/public/): Transfer failed for https://repository.cloudera.com/artifactory/public/net/sourceforge/czt/dev/cup-maven-plugin/1.6-cdh/cup-maven-plugin-1.6-cdh.pom: Unknown host repository.cloudera.com: Name or service not known

**Comment by dataalive** [comment]
try to change repo to aliyun ?

**Comment by sonysoul** [comment]
yes,I meet the same build failure with issue owner.I pull the docker image，then git doris repository,then sh build.sh --clean --be --fe ,and meet it. I also change maven 's mirror configuration , add aliyun mirror into it.
